src/components/tools/FrequencyCalculator.tsx

```diff
@@ -289,6 +289,33 @@ const modemPresets = new Map<
   ],
 ]);
 
+// Helper function to format the channel name (e.g., LONG_FAST -> LongFast)
+const formatChannelName = (name: string): string => {
+  return name
+    .toLowerCase()
+    .split("_")
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join("");
+};
```

Comment:
This won't be right. Many presets don't use something following this simple transformation.

The actual values are all in the switch statement at https://github.com/meshtastic/firmware/blob/master/src/DisplayFormatters.cpp#L5

Reply:
Okay, maybe not "many", looking again. But two of them, anyway, haha.